### PR TITLE
Feature input file merge

### PR DESCRIPTION
**Pull request by RoccoBombardieri**
Further attempt to comply with Travis' tests.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -3,7 +3,7 @@
 EIGEN_PATH = ./pyBeam/externals/Eigen
 CODI_PATH = ./pyBeam/externals/CoDiPack/include
 
-INCLPATH =  -I$(EIGEN_PATH) -I$(CODI_PATH) $(PYBEAM_INCLUDEPY)
```
Comment (rsanfer) [inline Makefile]: this shouldn't be necessary, the command `export PYBEAM_INCLUDEPY="$(python-config --includes)"` already incorporates the -I

File: pyBeam/include/FiniteElement_temp.h [hunk 2]
```diff
@@ -0,0 +1,77 @@
+/*
+ * pyBeam, a Beam Solver
```
Comment (rsanfer) [inline pyBeam/include/FiniteElement_temp.h]: What is this new file for?

File: tests/OneraM6/mesh_beam.msh~ [hunk 3]
```diff
@@ -0,0 +1,48 @@
+% Respect the format, mesh and conectivity have to start in line following NPOIN and NELEM
+
```
Comment (rsanfer) [inline tests/OneraM6/mesh_beam.msh~]: this file is copied

File: tests/regression/regression.py [hunk 4]
```diff
@@ -24,16 +25,64 @@
 #
 
 
-from pyBeam import CBeamSolver
+import pdb
```
Comment (rsanfer) [inline tests/regression/regression.py:28]: why is this pdb instead of pyBeam?

File: tests/regression/regression.py [hunk 5]
```diff
@@ -24,16 +25,64 @@
...
+import sys, os
+sys.path.append(str(os.path.realpath(__file__))[:-31] + '/pyBeam')
+sys.path.append(str(os.path.realpath(__file__))[:-31] + '/pyBeam/python')
+#sys.path.append('../../pyBeam')
+#sys.path.append('../../pyBeam/python')
+import in_out
+import swig
 
-beam = CBeamSolver()
+def Input_parsing(BEAM_config, inputs):
```
Comment (rsanfer) [inline tests/regression/regression.py]: This should become a generic function in the pyBeam library

**Review by rsanfer — CHANGES_REQUESTED** [review]
Hi Rocco,
although the tests pass (because I forgot to add an absolute value in the tolerance), you can see that the adjoint is broken in the job log, all the derivatives are 0. I'll do a quick fix to that, so the tests are safer.
I also did a couple of comments here and there, so you can have a look. In general it looks good, but there are a couple of things that you need to check. Also, we shouldn't include the binary files .pyc in the repository: use .gitignore to avoid them being tracked.

File: Makefile (hunk 1 above)
Comment (RoccoBombardieri) [reply Makefile]: Ok, I need to figure out how to do that

File: pyBeam/include/FiniteElement_temp.h (hunk 2 above)
Comment (RoccoBombardieri) [reply pyBeam/include/FiniteElement_temp.h]: temporary. It will be gone in the next version.

File: tests/OneraM6/mesh_beam.msh~ (hunk 3 above)
Comment (RoccoBombardieri) [reply tests/OneraM6/mesh_beam.msh~]: Yeah I will also delete al the temporary files from the repository

File: tests/regression/regression.py (hunk 4 above)
Comment (RoccoBombardieri) [reply tests/regression/regression.py:28]: Ok, this was a reorganisation of the code. pdb was just for debug. All the swigged classes are inside the folder swig. import swig does all the work

File: tests/regression/regression.py (hunk 5 above)
Comment (RoccoBombardieri) [reply tests/regression/regression.py]: I'll take care of that

**Pull request merged**